_data/changelogs/component-table.yml

```
@@ -4,6 +4,13 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Restore standard table and striped table component previews to guidance page
+    summaryAdditional:
+    affectsGuidance: true
+    githubPr: 2143
+    githubRepo: uswds-site
+    versionUswds: 3.6.0
```

This is a site only change right? This should be 3.5.0.

```suggestion
    versionUswds: 3.5.0
```

---

Resolved in https://github.com/uswds/uswds-site/pull/2143/commits/3903ed845aa65c19ad76f99de4f627cc3979d38e